cas-server-support-duo/src/main/java/com/duosecurity/duoweb/Util.java

```diff
@@ -7,6 +7,15 @@
 import javax.crypto.spec.SecretKeySpec;
 
 public class Util {
+    /**
+     * TODO: 12/10/15   move to DuoWeb
```

Comment:
A comment I didn't mean to commit. However, this function is only used by DuoWeb, so IDEA complained and I agreed. I'm going to be removing the rest of the functions in this util class replacing them w/ commons-codec functions. Commons-codec also has support for URL safe base64 encoding.